Apps/W1/Shopify/app/src/Payments/Enums/ShpfyDisputeType.Enum.al

```diff
@@ -0,0 +1,24 @@
+namespace Microsoft.Integration.Shopify;
+
+enum 30151 "Shpfy Dispute Type"
+{
+    /// <summary>
+    /// Enum Shpfy Dispute Type (ID 30151).
+    /// </summary>
+    /// 
+    Caption = 'Shopify Dispute Type';
+    Extensible = true;
```

Comment:
I don't think that this enum should be extensible.
I don't see that it supports more than the two values Inquiry and Chargeback.
https://shopify.dev/docs/api/admin-rest/2024-01/resources/dispute#resource-object